xmodule/video_block/video_block.py

```diff
@@ -188,18 +185,26 @@ def get_transcripts_for_student(self, transcripts, dest_lang=None):
         sorted_languages = OrderedDict(sorted_languages)
         return track_url, transcript_language, sorted_languages
 
+    def is_hls_playback_enabled(self, course_id):
+        """
+        Check if HLS playback is enabled for the course.
+        """
+        video_config_service = self.runtime.service(self, 'video_config')
+        return video_config_service.is_hls_playback_enabled(course_id) if video_config_service else False
+
     @property
     def youtube_deprecated(self):
         """
         Return True if youtube is deprecated and hls as primary playback is enabled else False
         """
+        video_config_service = self.runtime.service(self, 'video_config')
         # Return False if `hls` playback feature is disabled.
-        if not HLSPlaybackEnabledFlag.feature_enabled(self.location.course_key):
+        if video_config_service and not self.is_hls_playback_enabled(self.location.course_key):
```

Comment:
```suggestion
        if not self.is_hls_playback_enabled(self.location.course_key):
```

If the video service isn't available, we want to return False, right? ` self.is_hls_playback_enabled` checks for that.